### ISSUE TITLE
[Stripe SCA] Inform customers n days before their subscription orders are due payment if likely to need SCA authentication

### ISSUE THREAD

**Issue by kristinalim** [openfoodfoundation/openfoodnetwork#4183]
## Description

- Relates to #4170 

Email notification for customers likely to need SCA authentication _n_ days before their subscription orders are due payment. - I think this should be less than 7 days. Stripe releases charge authorization if not captured within 7 days.

* Do we need to create the Stripe `PaymentIntent` record for the subscription order/payment days before the payment is due, so we can inform the user earlier?
* Do we need to authorize payment?

**- As a:** customer
**- On page:** email
**- I want to be able to do:** Know in advance if I am likely to need to perform SCA authentication before payment is due for my subscription order

**Prerequisites:** #4182 and maybe #4181

## Acceptance Criteria

As a customer, I receive an email with info and instructions to perform SCA authentication days ahead of payment deadline for a subscription order if it was detected that I am likely to need SCA authentication.

**Comment by andrewpbrett** [comment]
@lin-d-hop I'd like to propose moving this out of the SCA epic and into All The Things. It feels like a nice-to-have if we're going to send an email to the customer if authorization is required anyway; doing it preemptively might just cause more confusion and add complexity, since technically the order could still be modified up until the OC closes. 

There's no way of telling ahead of time if a charge is likely to need SCA authorization without actually authorizing the charge and finding out, which then puts us in this limbo-like state where we'd want to hold on to the authorization but not actually run the charge... which is not something that we're set up to do, I believe, without a fair amount of additional work. So yeah. I'd bump this down the priority list.

**Comment by RachL** [comment]
@lin-d-hop @andrewpbrett are you ok if we close this one? I'm afraid it will just remain in all the things forever :D

**Comment by andrewpbrett** [comment]
That's fine with me.
